Automattic/simplenote-electron
# Update importers to use a single dialog

### Fix

First PR to look at #2570. This is starting to combine the importers to one Dialog so that any of the supported file types can be chosen and imported. This also now allows the ability to import more than one Simplenote or Evernote export file at the same time.

**Screenshots**
<details><summary>Light mode select files</summary>
<img width="867" alt="Screen Shot 2021-03-08 at 10 23 46 AM" src="https://user-images.githubusercontent.com/1326294/110337111-f17dbb80-7ffb-11eb-99f1-ed94ddf156d9.png">
</details>
<details><summary>Light mode files added</summary>
<img width="843" alt="Screen Shot 2021-03-17 at 12 20 50 PM" src="https://user-images.githubusercontent.com/1326294/111492592-90509900-871b-11eb-98d3-5aba35cd6818.png">
</details>
<details><summary>Light mode complete</summary>
<img width="845" alt="Screen Shot 2021-03-17 at 12 20 19 PM" src="https://user-images.githubusercontent.com/1326294/111492614-93e42000-871b-11eb-9c21-af99beb283f9.png">
</details>

<details><summary>Dark mode select files</summary>
<img width="832" alt="Screen Shot 2021-03-08 at 10 28 45 AM" src="https://user-images.githubusercontent.com/1326294/110337150-fb072380-7ffb-11eb-9bb2-b8d3be672e6a.png">
</details>
<details><summary>Dark mode files added</summary>
<img width="856" alt="Screen Shot 2021-03-17 at 12 19 22 PM" src="https://user-images.githubusercontent.com/1326294/111492497-7b740580-871b-11eb-833d-912dd92538ec.png">
</details>
<details><summary>Dark mode complete</summary>
<img width="851" alt="Screen Shot 2021-03-17 at 12 19 54 PM" src="https://user-images.githubusercontent.com/1326294/111492526-80d15000-871b-11eb-91a5-fe6e9162178f.png">
</details>


### Test
- Go to Settings > Tools > Import Notes.
- Notice you can select any supported files. 
- Choose multiple combinations of txt, md, Simplenote, and Evernote files.
- Ensure they import as expected.
- Ensure selecting to enable markdown for all notes does enable markdown.
- Smoke test in light and dark mode to ensure styling looks correct.

### Release
- Updating imports to a single dialog where any supported file type can be imported at the same time.
- Updating the styles of the import dialogs.
